lib/src/chat/chat.tsx

```diff
@@ -149,6 +149,10 @@ export const ChatInternal: FC<ChatProps> = (props: ChatProps) => {
   /**
    * Lifecycle: load one-off props
    */
+  useEffect(() => {
+    (pubnub as any)._config._addPnsdkSuffix('chat-components', 'RCC/__VERSION__');
```

Comment:
I know this is ugly as hell, but `_config` is not exposed in the types so we need to circumvent the type system here unfortunately.